### DISCUSSION TITLE
ASReview can't see my prior knowledge

### DISCUSSION THREAD

**Discussion by iacadamse** [asreview/asreview#626] [Troubleshooting (report errors in the "Issues" tracker)] [answered]
I uploaded my excel file to ASReview with a variable "Included". Within this variable I coded 0 for articles I want to exclude and 1 for articles I want to include. However when I upload my file ASReview does not see any of this prior knowledge. I get the following error:

CLI: ERROR:root:invalid literal for int() with base 10: '  '

How can I fix this?

**Comment by Rensvandeschoot** [comment, marked as answer]
Dear @iacadamse , thank you for raising this issue! It seems there is something present in a column where empty cells were expected. This is indicated between the brackets `' '` and it seems there are spaces probably in your column with historical labelling decisions. Can you check this, remove the spaces and try again?

**Reply by iacadamse** [reply]
This was it, thank you for helping me!
